lib/chromic_pdf/supervisor.ex

```diff
@@ -451,50 +454,37 @@ defmodule ChromicPDF.Supervisor do
       See [`print_header_footer_template.html`](https://cs.chromium.org/chromium/src/components/printing/resources/print_header_footer_template_page.html)
       from the Chromium sources to see how these values are interpreted.
 
-      ### Wait for dynamic content
+      ### Dynamic Content
 
-      If your HTML fetches dynamic content via JavaScript, you can declare the content as "ready" by setting an attribute on a DOM element. You can wait for this attribute with the `wait_for` option.
+      #### Evaluate script before printing
 
-      The following example waits for the DOM element with ID `#my-element` to receive the attribute `ready-to-print`
-      before printing the page:
+      In case your print source is generated by client-side scripts, for instance to render
+      graphics or load additional resources, you can trigger these by evaluating a JavaScript
+      expression before the PDF is printed.
 
-          wait_for = %{
-            selector: "#my-element",
-            attribute: "ready-to-print"
+          evaluate: %{
+            expression: """
+            document.querySelector('body').innerHTML = 'hello world';
+            """
```

Comment:
maybe it could even be nice to have some kind of helper to load `js.eex` templates, that we use ourselves :)